packages/prism-react-renderer/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "prism-react-renderer",
+  "version": "1.3.5",
+  "description": "Renders highlighted Prism output using React",
+  "sideEffects": true,
+  "main": "dist/index.js",
+  "module": "dist/index.mjs",
+  "types": "dist/index.d.ts",
+  "license": "MIT",
+  "repository": "https://github.com/FormidableLabs/prism-react-renderer",
+  "files": [
+    "index.d.ts",
+    "dist",
+    "prism",
+    "themes"
+  ],
+  "scripts": {
+    "postinstall": "pnpm run build:languages",
+    "prebuild": "patch-package",
```

Comment:
When this is ready for review we'll want to chat strategy on `prebuild` and `postinstall`.